.ci/install_kata_image_rust.sh

```diff
@@ -45,8 +45,10 @@ build_rust_image() {
 	echo "Building rust image"
 	sudo -E USE_DOCKER=1 DISTRO="${distro}" make -e image
 
-	echo "Install rust image to /usr/share/kata-containers"
-	sudo install "${GOPATH}/src/${osbuilder_repo}/kata-containers.img" "/usr/share/kata-containers/"
+	image_path="/usr/share/kata-containers/"
+	echo "Install rust image to ${image_path}"
+	sudo mkdir -p "${image_path}"
```

Comment:
I think the `-D` option to `install` can/will do that for you?
/cc @jodh-intel

Reply:
+1

Reply:
@chavafg and @grahamwhaley changes applied